controllers/domain-redirect.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 module.exports = function(req, res, next) {
-    if (req.get('host') === 'www.biglotteryfund.org.uk') {
-        res.redirect(301, `https://www.tnlcommunityfund.org.uk${req.originalUrl}`);
+    if (req.get('host') === 'apply.tnlcommunityfund.org.uk') {
+        res.redirect(301, `https://www.tnlcommunityfund.org.uk/apply`);
```

Comment:
Thought this made more sense than trying to preserve the original path which would always 404.